src/nimqml/private/dotherside.nim

```diff
@@ -371,6 +371,12 @@ proc dos_qsettings_remove(vptr: DosQSettings, key: cstring)
   {.cdecl, dynlib: dynLibName, importc.}
 proc dos_qsettings_delete(vptr: DosQSettings) 
   {.cdecl, dynlib: dynLibName, importc.}
+proc dos_qsettings_begin_group(vptr: DosQSettings, group: cstring) 
+  {.cdecl, dynlib: dynLibName, importc.}
+proc dos_qsettings_end_group(vptr: DosQSettings) 
+  {.cdecl, dynlib: dynLibName, importc.}
+proc dos_qsettings_child_keys(vptr: DosQSettings) 
```

Comment:
Hmm, I did not see this one in DOtherSide ?

Reply:
Great catch.